python/taichi/main.py

```diff
@@ -854,13 +854,11 @@ def debug(self, arguments: list = sys.argv[2:]):
         # Short circuit for testing
         if self.test_mode: return args
 
+        os.environ['TI_DEBUG'] = '1'
+        os.environ['TI_LOG_LEVEL'] = 'debug'
         ti.core.set_core_trigger_gdb_when_crash(True)
 
-        with open(args.filename) as script:
-            script = script.read()
-
-        # FIXME: exec is a security risk here!
-        exec(script, {'__name__': '__main__'})
+        runpy.run_path(args.filename, run_name='__main__')
```

Comment:
@rexwangcc `ti debug` was not working correct, I fixed this, am I fixing correct?